test/tests-scheduling.cpp

```diff
@@ -845,9 +845,9 @@ TEST(scheduling_eval_test, spmv_fuse) {
           .parallelize(block, ParallelUnit::GPUBlock, OutputRaceStrategy::IgnoreRaces)
           .parallelize(warp, ParallelUnit::GPUWarp, OutputRaceStrategy::Atomics)
           .parallelize(thread, ParallelUnit::GPUThread, OutputRaceStrategy::Atomics);
-//  ir::CodeGen_CUDA codegen = ir::CodeGen_CUDA(cout, ir::CodeGen_CUDA::ImplementationGen);
-//  ir::Stmt compute = lower(stmt, "compute",  false, true);
-//  codegen.print(compute);
+  ir::CodeGen_CUDA codegen = ir::CodeGen_CUDA(cout, ir::CodeGen_CUDA::ImplementationGen);
```

Comment:
What is this for?

Reply:
This was Ryan's method of testing what the CUDA code generation would print out in terminal when you run a googletest ctest. I can comment it back out (which I should probably do).